### ISSUE TITLE
Output TPM2.net protocol over serial

### ISSUE THREAD

**Issue by HakanL** [OpenLightingProject/ola#556]
So we don't have to go through java code like this: https://github.com/neophob/ola-to-tpm2net

but are able to get full pixel data over a serial port. It's important that we support more than 512 channels (multiple universes) though, 170 pixels don't last that long.

**Comment by peternewman** [comment]
There is an English description here, which I assume is the same?
https://gist.github.com/jblang/89e24e2655be6c463c56

**Comment by peternewman** [comment]
Similar to Stageprofi, we probably ideally want serial and network support, of TPM2 and TPM2.net.

**Comment by HakanL** [comment]
Yes, that gist for TPM2.Net in English is also what I found as the de-factor specification for the TPM2.Net protocol.
